gallery/src/styles/NxList/NxListClickableExample.tsx

```diff
@@ -10,31 +10,41 @@ import { NxFontAwesomeIcon } from '@sonatype/react-shared-components';
 
 const NxListClickableExample = () =>
   <ul className="nx-list nx-list--clickable">
-    <li className="nx-list__item" tabIndex={0}>
-      <span className="nx-list__text">Action 1</span>
-      <NxFontAwesomeIcon icon={faAngleRight} className="nx-chevron" />
+    <li className="nx-list__item nx-list__item--link">
```

Comment:
I feel like the `.nx-list__item--link` class name should be changed if it's being used for things other than links.  Probably `.nx-list__item--clickable`.  Of course we'll still need to keep `.nx-list__item--link` for backwards compat too

Reply:
> I feel like the `.nx-list__item--link` class name should be changed if it's being used for things other than links. Probably `.nx-list__item--clickable`. Of course we'll still need to keep `.nx-list__item--link` for backwards compat too

Addressed here: https://github.com/sonatype/sonatype-react-shared-components/pull/226/commits/035f4534eb8c5c58eaa029563f146b39d9acc559